src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -1025,12 +1022,12 @@ namespace WindowEquivalentLayer {
         QAllSWwinAbs({1, NL + 1}) = QRadSWwinAbs({1, NL + 1}, SurfNum);
         //  Solve energy balance(s) for temperature at each node/layer and
         //  heat flux, including components, between each pair of nodes/layers
-        ASHWAT_ThermalR = ASHWAT_Thermal(
-            CFS(EQLNum), TIN, Tout, HcIn, HcOut, TRMOUT, TRMIN, 0.0, QAllSWwinAbs({1, NL + 1}), TOL, QOCF, QOCFRoom, T, Q, JF, JB, H, UCG, SHGC);
-        // long wave radiant power to room not including reflected
-        QRLWX = JB(NL) - (1.0 - LWAbsIn) * JF(NL + 1);
-        // nominal surface temp = effective radiant temperature
-        SurfInsideTemp = TRadC(QRLWX, LWAbsIn);
+        ASHWAT_ThermalR =
+            ASHWAT_Thermal(CFS(EQLNum), TIN, Tout, HcIn, HcOut, TRMOUT, TRMIN, 0.0, QAllSWwinAbs({1, NL + 1}), TOL, QOCF, QOCFRoom, T, Q, JF, JB, H);
```

Comment:
Two arguments gone.  OK, I like that though I am curious why they were needed before but aren't now.

Reply:
These two variables (UCG and SHGC) are rated window performance values. They do not need to be calculated for each time step. That is why I made them optional arguments.